src/components/dialog/Body.tsx

```diff
@@ -1,8 +1,9 @@
-import React, { ReactNode, forwardRef } from 'react';
+import React, { ReactNode, forwardRef, RefObject } from 'react';
 
 import { Box } from '../../index';
+import { BoxProps } from '../box/Box';
 
-interface BodyProps {
+interface BodyProps extends Omit<BoxProps, 'ref'> {
```

Comment:
I would type the ref explicitly instead of extending from Box, since it's not always a Box, but can also be a div

Reply:
Thats what we're doing now right? We're extending everything from BoxProps except ref, and then do `as RefObject<HTMLDivElement>` on the div's ref

Reply:
Ah yes you're right, I got confused 🙈